site/scripts/prepare-blocks.ts

```diff
@@ -154,37 +148,42 @@ const ensureRepositorySnapshot = async ({
     return repositorySnapshotDirPath;
   }
 
-  const { path: unzipDirPath, cleanup: cleanupUnzipDirPath } = await tmp.dir({
+  const { path: untarDirPath, cleanup: cleanupUntarDirPath } = await tmp.dir({
     unsafeCleanup: true,
   });
 
   try {
-    console.log(chalk.green(`Downloading ${zipUrl}...`));
-    // @todo Consider finding cross-platform NPM packages for curl and unzip commands
+    console.log(chalk.green(`Downloading ${tarballUrl}...`));
+    // @todo Consider finding cross-platform NPM packages for curl and untar commands
     await execa(
       "curl",
-      ["-sL", "-o", path.resolve(unzipDirPath, `repo.zip`), zipUrl],
+      ["-sL", "-o", path.resolve(untarDirPath, `repo.tar.gz`), tarballUrl],
       defaultExecaOptions,
     );
 
+    const untarPath = path.resolve(untarDirPath, "repo");
+
+    console.log(chalk.green(`Creating output directory...`));
+    await execa("mkdir", ["-p", untarPath]);
```

Comment:
Since `unzip` doesn't create the output directory for us, we do that manually here. The `-p` flag allows us to do it in an idempotent way.

Reply:
We don’t need `execa` here:


```suggestion
    await fs.ensureDir(...)
```

https://github.com/jprichardson/node-fs-extra/blob/HEAD/docs/ensureDir.md